Vonk.Plugin.DocumentOperation/DocumentService.cs

```diff
@@ -253,14 +258,24 @@ private void CancelDocumentOperation(IVonkResponse response, int statusCode, Iss
 
         #endregion Helper - Return response
 
-        private IssueComponent LocalReferenceNotResolvedIssue(string failedReference)
+        private IssueComponent ReferenceNotResolvedIssue(string failedReference)
         {
             var issue = new OperationOutcome.IssueComponent()
             {
-                Code = IssueType.NotFound,
                 Severity = IssueSeverity.Error,
-                Details = new CodeableConcept("http://hl7.org/fhir/ValueSet/operation-outcome", "MSG_LOCAL_FAIL", "Unable to resolve local reference to resource " + failedReference)
             };
+
+            if (IsRelativeUrl(failedReference))
+            {
+                issue.Code = IssueType.NotFound;
+                issue.Details = new CodeableConcept("http://hl7.org/fhir/ValueSet/operation-outcome", "MSG_LOCAL_FAIL", "Unable to resolve local reference to resource " + failedReference);
+            }
+            else
```

Comment:
This 'else' can be directly in ResolveResource, so you don't have to check  IsRelativeReference twice.